index.html

```diff
@@ -203,105 +203,123 @@ <h2>Multi-Post <br>
-                            </div>
-                            <div class="separator">
-                                <img src="img/separator.png" alt="separator">
-                            </div>
-                            <div class="language-title">
-                                <div class="title1">Skills</div>
-                                <div class="arrow1">
-                                    <img src="img/rightArrow.png" alt="right arrow">
+                            </li>
+                            <li class="list-flex ">
+                                <a>
+                                    <img src="img/html.png" alt="Html">
+                                </a>
+                                <div class="name-prom">
+                                    Html
+                                </div>
+                            </li>
+                            <li class="list-flex ">
+                                <a>
+                                    <img src="img/css.png" alt="CSS">
+                                </a>
+                                <div class="name-prom">
+                                    Css
                                 </div>
+                            </li>
+                        </ul>
+                        <div class="language-title">
+                            <div class="title1">Frameworks</div>
+                            <div class="arrow">
+                                <img src="img/rightArrow.png" alt="right arrow">
                             </div>
-                            <div class="separator1">
-                                <img src="img/separator.png" alt="separator">
+                        </div>
+                        <div class="separator">
+                            <img src="img/separator.png" alt="separator">
+                        </div>
+                        <div class="language-title">
+                            <div class="title1">Skills</div>
+                            <div class="arrow1">
+                                <img src="img/rightArrow.png" alt="right arrow">
                             </div>
                         </div>
-                    </li>
-                </ul>
-            </div>
+                        <div class="separator1">
+                            <img src="img/separator.png" alt="separator">
+                        </div>
+                    </div>
+                </li>
+            </ul>
         </div>
+        <section class="contact">
+            <article class="contact_footer">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested
+                    in developing, a feature that you need built or a project
+                    that needs coding. <br>I'd
+                    love to help with it!</p>
+                <form class="form-start" action="https://formspree.io/f/xlekpydn" method="POST">
```

Comment:
- [ ] There is currently an overall design defect coming from your application on mobile view, there is margin to the right of your webpage as shown in the image below for your design, this can be as a result of excess margin or margin please kindly look into the design for your application and fix accordingly. :+1: 
- [ ] The form as shown in the image below is not consistent with the figma design, your form is currently overflowing the viewport and it is not centered, this can be as a result of using fixed width in `px`, i will kindly suggest you provide particular styling for thr form width in `%` and then center it in the page using `marginLauto` :+1: 
- figma design
![f-d-s](https://user-images.githubusercontent.com/89055502/236348998-141da815-474c-450b-a5e4-7e3741d96dad.png)
- your design
![v-d-s](https://user-images.githubusercontent.com/89055502/236349021-8b6d1dec-a14f-4d01-bb66-fba1556fabd1.png)